packages/libsql-client/examples/sync.js

```diff
@@ -1,32 +1,37 @@
 import { createClient } from "@libsql/client";
-import reader from "readline-sync";
```

Comment:
This is an example for people who want to understand how to use the API.

Can you please make a copy of this for the test case but keep the original as-is?